src/MongoDB.Driver.Core/TransactionOptions.cs

```diff
@@ -26,6 +27,39 @@ namespace MongoDB.Driver
     /// </summary>
     public class TransactionOptions
```

Comment:
Do you typically add unit tests for classes like this?

Reply:
Done